install/mod_pagespeed_test/unauthorized/inline_css.html

```diff
@@ -1,7 +1,7 @@
 <html>
   <head>
     <title>inline_css example</title>
-    <link rel="stylesheet" href="http://cse.google.com/cse/style/look/default.css">
+    <link rel="stylesheet" href="http://modpagespeed.com/testfiles/google-cse-default.css">
```

Comment:
yes, my idea was that `modpagespeed.com` is just as unauthorized in the tests as `cse.google.com`. I'll add a comment